php/WP_CLI/Runner.php

```diff
@@ -334,6 +334,44 @@ private function _run_command() {
 	}
 
 	/**
+	 * Perform a command against a remote server over SSH
+	 */
+	private function run_ssh_command( $ssh ) {
+
+		$host = $ssh;
+		$path = '';
+		if ( false !== ( $key = stripos( $host, ':' ) ) ) {
+			$path = substr( $host, $key + 1 );
+			$host = substr( $host, 0, $key );
+		}
+
+		WP_CLI::debug( 'SSH host: ' . $host, 'bootstrap' );
+		WP_CLI::debug( 'SSH path: ' . $path, 'bootstrap' );
+
+		$is_tty = function_exists( 'posix_isatty' ) && posix_isatty( STDOUT );
+
+		$wp_binary = 'wp';
```

Comment:
> No, I didn't. Also, perhaps the PHAR isn't even aliased as wp on the target environment. I think that there will ultimately need to be a way to provide an alternate path.

I ended up creating a `WP_CLI_PRE_SSH_CMD` env variable, to offer a bit more flexibility:

```
WP_CLI::add_hook( 'before_ssh', function() {
	if ( $ssh = WP_CLI::get_runner()->config['ssh'] ) {
		list( $host, $path ) = explode( ':', $ssh );
		switch( $host ) {
			case 'hb':
				putenv( 'WP_CLI_SSH_PRE_CMD=export PATH=$HOME/bin:$PATH' );
				break;
		}
	}
});
```